FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,57 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImageMapper.map(data, from: response))
+			case .failure(_):
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+private final class FeedImageMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+
+		var imageFeed: [FeedImage] {
```

Comment:
Since `FeedImageMapper.Root` is private then it is clear that `feed` would be about feed images. Thus the property name can be shorter, just `feed`

Please rename property to `feed`:
```suggestion
		var feed: [FeedImage] {
```